src/turnkeyml/cli/cli.py

```diff
@@ -125,9 +131,11 @@ def main():
         "-i",
         "--input-files",
         nargs="+",
-        help="One or more script (.py), ONNX (.onnx), or input list (.txt) files to be benchmarked",
+        help="One or more inputs that will be evaluated by the tool sequence "
+        "(e.g., script (.py), ONNX (.onnx), turnkey build state (state.yaml), "
+        "input list (.txt) files)",
         type=lambda file: _check_extension(
-            ("py", "onnx", "txt"), file, parser.error, tool_classes
+            ("py", "onnx", "txt", "yaml"), file, parser.error, tool_classes
```

Comment:
Sometimes yaml files get saved as `.yml` files. Should we check for .yml as well?

Reply:
We only support state.yaml files - this is just a very early check for correct file extensions. The "yaml" in "state.yaml" is hardcoded, so we actually know the input is illegal if it is ".yml".